openbas-front/src/components/common/pagination/PaginationComponent.tsx

```diff
@@ -41,6 +41,7 @@ const useStyles = makeStyles(() => ({
 const ROWS_PER_PAGE_OPTIONS = [20, 50, 100];
 
 interface Props<T> {
+  key?: React.Key;
```

Comment:
This new param seems to not be used

Reply:
This param is used in Scenarios.tsx to force the pagination to refresh in the case of a duplication or deletion of an element from the list

Reply:
Ok but the parameter is not used in the class. It looks like magic.

And by making this change, we bypass the use of the store.

What is the use case?